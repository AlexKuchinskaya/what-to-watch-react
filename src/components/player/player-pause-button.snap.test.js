import React from 'react';
import {render} from '@testing-library/react';
import PlayerPauseButtonSvg from './player-pause-button';

it(`Should PlayerPauseButton render correctly`, () => {
  const {container} = render(<PlayerPauseButtonSvg />);
  expect(container).toMatchSnapshot();
});
