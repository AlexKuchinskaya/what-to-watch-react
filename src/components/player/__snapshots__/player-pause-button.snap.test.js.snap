// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PlayerPauseButton render correctly 1`] = `
<div>
  <svg
    height="21"
    viewBox="0 0 14 21"
    width="14"
  >
    <use
      xlink:href="#pause"
    />
  </svg>
</div>
`;
